### ISSUE TITLE
Allow setting for accuracy when saving track

### ISSUE THREAD

**Issue by JensKoopmann** [merlos/iOS-Open-GPX-Tracker#244]
Since the accuracy of the gps position is already displayed, it would be great to also use it to avoid inaccurate data in the track

**Comment by vincentneo** [comment]
I assume this is a duplicate of #155?

**Comment by JensKoopmann** [comment]
Not exactly. My suggestion is not to save to information since it would not be a standard gpx field but to only save points with an sufficient accuracy that can be set by the user
